app/javascript/packages/form-link/package.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "@18f/identity-form-link",
+  "version": "1.0.0",
+  "private": true
```

This will now cause build failures on `main` since #9961 was merged. I'd suggest rebasing or merging in the latest main.

```suggestion
  "private": true,
  "sideEffects": [
    "./form-link-element.ts"
  ]
```